docs/source/user-guide/example-usage.md

```diff
@@ -175,12 +175,13 @@ codegen-units = 1
 
 Then, in `main.rs.` update the memory allocator with the below after your imports:
 
-```rust
+```rust,ignore
```

Comment:
Does it need add blank between them?

Reply:
I tried it locally, and it doesn't need a blank line. I'll enable doctest CI as well in the next commit